vnext/Microsoft.ReactNative/XamlUIService.idl

```diff
@@ -19,6 +19,7 @@ namespace Microsoft.ReactNative
     DOC_STRING("Use this method to get access to the @XamlUIService associated with the @IReactContext.")
     static XamlUIService FromContext(IReactContext context);
 
+#ifndef USE_FABRIC
```

Comment:
I know this is just a starting place..  But we probably want this whole XamlUIService class to not existing when !USE_FABRIC

Reply:
I started with that, but too much needs it, especially as you (used to?) need the SetXamlRoot for the islands APIs. i wasn't sure if it was going to mess up the current islands work

Reply:
Interesting.  - but yeah we can do further cleanup in subsequent check-ins.
We shouldn't need XamlRoot anymore. - Any Xaml requirements I want to move to react-native-xaml in the future.